spec/support/shared_examples/infinite_scroll.rb

```diff
@@ -0,0 +1,25 @@
+
+shared_examples 'infinite scroll' do
+  it do
```

Comment:
I prefer to use many examples instead of only it, and also I prefer to use named examples :disappointed:

Reply:
:grimacing: